script/Propose.s.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.17;
+
+import {Script} from "forge-std/Script.sol";
+import {GitcoinGovernor} from "src/GitcoinGovernor.sol";
+import {IGovernorAlpha} from "src/IGovernorAlpha.sol";
+import {ICompoundTimelock} from "openzeppelin-contracts/governance/extensions/GovernorTimelockCompound.sol";
+
+contract ProposeScript is Script {
+  IGovernorAlpha constant governorAlpha = IGovernorAlpha(0xDbD27635A534A3d3169Ef0498beB56Fb9c937489);
+  address constant proposer = 0xc2E2B715d9e302947Ec7e312fd2384b5a1296099; // kbw.eth
+
+  function propose(GitcoinGovernor _newGovernor) internal returns (uint256 _proposalId) {
```

Comment:
Hmm, let me think about it. I think what will end up happening is that we first Deploy the new Governor, then modify this script (and tests) to point to the already deployed version. So I think this will end up being overkill.

Reply:
Feel free to resolve this comment yourself